ztp/gitops-subscriptions/argocd/resource-hook-example/siteconfig/test-3node.yaml

```diff
@@ -0,0 +1,155 @@
+# test-node[123]-bmh-secret & assisted-deployment-pull-secret need to be created under same namespace test-3node
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-3node
+  labels:
+    name: test-3node
+---
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "test-3node"
+  namespace: "test-3node"
+spec:
+  baseDomain: "example.com"
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "openshift-4.10"
+  sshPublicKey: "ssh-rsa AAAA..."
+  clusters:
+  - clusterName: "test-3node"
+    clusterType: "standard"
+    clusterProfile: "du"
+    numMasters: 3
+    clusterLabels:
+      group-du-sno: ""
```

Comment:
This points out that our original naming of the group wasn't quite right. We use the group to combine features that we expect are more regionally specific (eg remote logging server). Using "du-sno" as part of the group name implies it is specific to that type of cluster, so when we re-use it here it looks "wrong". We should consider revisiting the group naming in subsequent work.

Reply:
Fair point.  Maybe for now I'll fix this just as "group-du" so it implies how you ought to have a different group policy for non-SNO clusters.